src/ui/qgswmssourceselectbase.ui

```diff
@@ -495,6 +488,20 @@
      </widget>
     </widget>
    </item>
+   <item row="4" column="0">
+    <widget class="QDialogButtonBox" name="buttonBox">
+     <property name="standardButtons">
+      <set>QDialogButtonBox::Help</set>
+     </property>
+    </widget>
+   </item>
+   <item row="3" column="0">
+    <widget class="QCheckBox" name="mLoadLayersIndividuallyCheckBox">
+     <property name="text">
+      <string>Load layers individually</string>
```

Comment:
How about:

```suggestion
      <string>Load as separate layers</string>
```

? To me that's a bit clearer. This could do with a nice explanatory tooltip too